Common/cpp/LayoutAnimations/LayoutAnimationsManager.cpp

```diff
@@ -94,6 +94,27 @@ void LayoutAnimationsManager::startLayoutAnimation(
       config->getJSValue(rt));
 }
 
+void LayoutAnimationsManager::cancelLayoutAnimation(
+    jsi::Runtime &rt,
+    int tag,
+    const std::string &type,
+    bool cancelled = true,
+    bool removeView = true) {
+  auto lock = std::unique_lock<std::mutex>(animationsMutex_);
+  jsi::Value layoutAnimationRepositoryAsValue =
+      rt.global()
+          .getPropertyAsObject(rt, "global")
+          .getProperty(rt, "LayoutAnimationsManager");
+  jsi::Function cancelLayoutAnimation =
+      layoutAnimationRepositoryAsValue.getObject(rt).getPropertyAsFunction(
+          rt, "stop");
+  auto config = sharedTransitionAnimations_[tag];
```

Comment:
I think we only need the lock for this line so maybe let's keep it scoped:
```suggestion
  {
    auto lock = std::unique_lock<std::mutex>(animationsMutex_);
    auto config = sharedTransitionAnimations_[tag];
  }
```